s3parq/publish_redshift.py

```diff
@@ -48,7 +145,7 @@ def _datatype_mapper(columns: dict) -> dict:
         else:
             raise ValueError(f"Error: {dtype} is not a datatype which can be mapped to Redshift.")
         sql_statement += f'{col} {dtypes[col]}, '
-    return "(" + sql_statement[:-2] + ")"
+    return f"({sql_statement[:-2]})"
```

Comment:
maybe add a short inline comment explaining why you remove the last two chars from the string?